frontend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-alpine
+FROM node:14-alpine
```

Comment:
I'm curious about the reason to make this change.

Reply:
The version of `node-sass` specified in the `package.json` in the `devDependencies` requires Node 14, according to the [documentation](https://www.npmjs.com/package/node-sass). Most likely `node:lts` was at version 14 until recently, but the now `node:lts-alpine` points to version version 16, according to [Dockerhub](https://hub.docker.com/_/node/).

Reply:
I see. Thanks for the explanation.